dev/com.ibm.ws.anno/resources/com/ibm/ws/anno/resources/internal/AnnoMessages.nlsprops

```diff
@@ -421,22 +421,22 @@ ANNO_TARGETS_CORRUPT_CLASS.useraction=The class can be replaced, removed or this
 # # {2} The error message from the exception
 ANNO_CLASSSOURCE_CHILD_OPEN_EXCEPTION=CWWKC0065W: [ {0} ]: The open of child class source [{1}] caused an exception. Error message: {2}
 ANNO_CLASSSOURCE_CHILD_OPEN_EXCEPTION.explanation=The open of the child class source caused an exception.
-ANNO_CLASSSOURCE_CHILD_OPEN_EXCEPTION.useraction=See the exception and the associated FFDC message for more information.
+ANNO_CLASSSOURCE_CHILD_OPEN_EXCEPTION.useraction=For more information on the topic, see the exception and the associated FFDC message.
 
 # # {0} A hashcode ID that identifies the object emitting the message
 # # {1} A hashcode ID that identifies the object causing the exception
 # # {2} The error message from the exception
 ANNO_CLASSSOURCE_CHILD_CLOSE_EXCEPTION=CWWKC0066W: [ {0} ]: The close of child class source [{1}] failed. Error message: {3}
 ANNO_CLASSSOURCE_CHILD_CLOSE_EXCEPTION.explanation=The close of the child class source caused an exception.
-ANNO_CLASSSOURCE_CHILD_CLOSE_EXCEPTION.useraction=See the exception and the associated FFDC message for more information.
+ANNO_CLASSSOURCE_CHILD_CLOSE_EXCEPTION.useraction=For more information on the topic, see the exception and the associated FFDC message.
 
 # # {0} A hashcode id used to identify the object emitting the message
 # # {1} An entry name
 # # {2} A class name
 # # {3} The error message from the exception
 ANNO_CLASSSOURCE_ENTRY_SCAN_EXCEPTION=CWWKC0067W: [ {0} ] The processing of entry [{1}] for class [{2}] caused an exception. Error message: {3}
 ANNO_CLASSSOURCE_ENTRY_SCAN_EXCEPTION.explanation=The processing of class information read from an entry caused an exception.
-ANNO_CLASSSOURCE_ENTRY_SCAN_EXCEPTION.useraction=See the exception and the associated FFDC message for more information.
+ANNO_CLASSSOURCE_ENTRY_SCAN_EXCEPTION.useraction=For more information on the topic, see the exception and the associated FFDC message.
```

Comment:
Change
'For more information on the topic, see the exception and the associated FFDC message.'
to
'For more information, see the exception and the associated FFDC message.'

Reply:
Updated